index.js

```diff
@@ -54,7 +75,26 @@ function getHighestMetascore() {}
  *  getAverageIMDBRating(movies);
  *  //> 7.76
  */
-function getAverageIMDBRating() {}
+function getAverageIMDBRating(movies) {
+  //console.log(movies.length);
+  if (!movies.length) {
+    return 0;
+  }
+  let imdbSum = 0;
+  let scoreArray = [];
+  for (let film of movies) {
+    for (source of film.ratings) {
+      if (source.source == "Internet Movie Database") {
+        let currentScore = source.value.split("/", 1);
+        scoreArray.push(currentScore);
```

Comment:
This is great logic - line 88 is really nice.  However, do we need to go to the trouble of pushing the scores we isolate into an array?  Can we simply add those each score like we do in 94 if we find a score here?  This would save us a final loop.